assets/css/screenplay.scss

```diff
@@ -1,3 +1,7 @@
+$theme-colors: (
+  "primary": #384956,
+);
```

Comment:
I don't think this is being used anymore? Not sure

Reply:
While it isn't being explicitly used by us, bootstrap uses it to calculate colors of element hover, focus, borders, etc. So I think it's good to keep.

Reply:
Gotcha. Can the color itself go into the colors file?

Reply:
I was taking the whole "override colors before imports" a little too literally. These overrides just need to be above the bootstrap import. If I import `variables.scss` first, we can use it like we do everywhere else.